From 914507aa6980129fefb4decaabea1a4f60c7182b Mon Sep 17 00:00:00 2001
From: Steve Dickson <steved@redhat.com>
Date: Wed, 21 Aug 2013 14:40:22 -0400
Subject: [PATCH 1/1] rpcbind: rpcuser not being set in Makefile.am

Upstream-Status: backport

Commit 8d7a0708 cause a regression where the rpcuser id was not
being set, which in turn cause rpcbind to immediately exit.
This patch removes the extra ',' that was in the AC_ARG_WITH
statement in the configure.ac file.

Signed-off-by: Steve Dickson <steved@redhat.com>
Signed-off-by: Cristian Iorga <cristian.iorga@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2b67720..1cf42d3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@ AC_ARG_WITH([statedir],
 AC_SUBST([statedir], [$with_statedir])
 
 AC_ARG_WITH([rpcuser],
-  AS_HELP_STRING([--with-rpcuser=ARG], [use ARG for RPC @<:@default=root@:>@]),
+  AS_HELP_STRING([--with-rpcuser=ARG], [use ARG for RPC @<:@default=root@:>@])
   ,, [with_rpcuser=root])
 AC_SUBST([rpcuser], [$with_rpcuser])
  
-- 
1.8.3.2

